app/src/main/java/com/example/android/service/BackendClient.java

```diff
@@ -1,16 +1,23 @@
 package com.example.android.service;
 
+import org.json.JSONObject;
+
 import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
```

Comment:
Could you please help remove unused imports?

```suggestion
import java.io.IOException;
```